include/openthread/link_metrics.h

```diff
@@ -115,6 +152,48 @@ otError otLinkMetricsQuery(otInstance *                aInstance,
                            otLinkMetricsReportCallback aCallback,
                            void *                      aCallbackContext);
 
+/**
+ * This function sends an MLE Link Metrics Management Request to configure/clear a Forward Tracking Series.
+ *
+ * @param[in] aInstance          A pointer to an OpenThread instance.
+ * @param[in] aDestination       A pointer to the destination address.
+ * @param[in] aSeriesId          The Series ID to operate with.
+ * @param[in] aSeriesFlags       A pointer to the Series Flags that specifies which frames are to be accounted. The
+ * pointer MUST NOT be `nullptr`.
+ * @param[in] aLinkMetricsFlags  A pointer to flags specifying what metrics to query. Should be `NULL` when
+ * `aSeriesFlags` is `NULL`.
```

Comment:
conflict - whether or not `aSeriesFlags` could be NULL? 
From the context, `aLinkMetricsFlags` should be `nullptr` when `@p aSeriesFlags` is a pointer to a Series Flags with value 0.

Reply:
Sorry I made a mistake. It shouldn't be `NULL` anytime. The comment actually means `aSeriesFlags` is `0`. I agree with your previous comment to pass `otLinkMetricsSeriesFlags` in this method instead of a pointer so that it could not be `NULL`.